dev-lang/erlang/files/epmd.init-r1

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the Erlang Public License 1.1
+
+ZT_ADDRESS=""
+pidfile="/var/run/epmd.pid"
+command_args="${command_args:--daemon -relaxed_command_check -address 127.0.0.1}"
```

Comment:
Why do you both install conf.d and supply a fallback? That doesn't seem to make much sense.